manifests/exports_data.pp

```diff
@@ -0,0 +1,20 @@
+define nfs::exports_data (
+  $config_title = "$title",
```

Comment:
I think you mean to use $name here. We don't need another variable just to change the name of the variables around.